### PR TITLE
Update DB setup instructions in the readme file.

### PR DESCRIPTION
**Pull request by mareksuscak**
I have noticed that there’s a mix alias that helps set up the database incl. seeds which might be helpful.

### CODE REVIEW

File: installer/templates/phx_umbrella/apps/app_name_web/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 To start your Phoenix app:
 
   * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
+  * Setup your database with `mix ecto.setup`
```
Comment (wojtekmach) [inline installer/templates/phx_umbrella/apps/app_name_web/README.md]: in umbrella, `ecto.setup` alias is defined in the domain logic app and aliases aren't shared between apps, so this does not work out of the box

Comment (mareksuscak) [inline installer/templates/phx_umbrella/apps/app_name_web/README.md]: oh so what do you suggest then?

Comment (wojtekmach) [inline installer/templates/phx_umbrella/apps/app_name_web/README.md]: Yeah, I should have written something. Honestly I don't know what's the best answer here, I'm sure maintainers will chime in. I like the `ecto.setup` change in non-umbrella project though!

**Comment by chrismccord** [comment]
Thanks! I would prefer to to keep the existing instructions since they apply to all applications. It also introduces users to tasks they will necessarily need to know about. `mix ecto.setup` is a convenient alias, but I believe the current instructions are best for now. Thanks!

**Pull request closed without merging**